src/CoreShop/Bundle/CoreBundle/EventListener/NotificationRules/CustomerListener.php

```diff
@@ -164,6 +164,7 @@ private function prepareCustomerParameters(CustomerInterface $customer)
             'lastname' => $customer->getLastname(),
             'email' => $customer->getEmail(),
             'object' => $customer,
+            'store' => $this->shopperContext->getStore(),
```

Comment:
I think we should check if we even have a shop here with
```
$this->shopperContext->hasStore()
```

@solverat What do think?

Reply:
@dpfaffenbauer I didn't notice that getStore throws an exception. I changed it to hasStore->getStore version.